Make the build system use $target_os and consider Hurd to be Linux
 reg. htop_platform
Source: https://patch-diff.githubusercontent.com/raw/hishamhm/htop/pull/472.patch
From: James Clarke <jrtc27@jrtc27.com>
Date: Mon, 18 Apr 2016 23:55:55 +0100
Subject: [PATCH 1/2] Use $target_os instead of $target in configure.ac

---
 configure.ac | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/configure.ac b/configure.ac
index fa32359..8d48d68 100644
--- a/configure.ac
+++ b/configure.ac
@@ -26,17 +26,17 @@ AC_PROG_LIBTOOL
 
 # Checks for platform.
 # ----------------------------------------------------------------------
-case "$target" in
-*linux*)
+case "$target_os" in
+linux*)
    my_htop_platform=linux
    ;;
-*freebsd*)
+freebsd*|kfreebsd*)
    my_htop_platform=freebsd
    ;;
-*openbsd*)
+openbsd*)
    my_htop_platform=openbsd
    ;;
-*darwin*)
+darwin*)
    my_htop_platform=darwin
    ;;
 *)

From 2de52862a693327e9d18057b175e4a7753a492e3 Mon Sep 17 00:00:00 2001
From: James Clarke <jrtc27@jrtc27.com>
Date: Mon, 18 Apr 2016 23:57:30 +0100
Subject: [PATCH 2/2] Use Linux backend on the Hurd

---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 8d48d68..ccbfb52 100644
--- a/configure.ac
+++ b/configure.ac
@@ -27,7 +27,7 @@ AC_PROG_LIBTOOL
 # Checks for platform.
 # ----------------------------------------------------------------------
 case "$target_os" in
-linux*)
+linux*|gnu*)
    my_htop_platform=linux
    ;;
 freebsd*|kfreebsd*)
